sparsity/test/test_dask_sparse_frame.py

```diff
@@ -63,29 +62,23 @@ def test_loc(iindexer, correct_shape):
     assert res.shape == correct_shape
 
 def test_dask_loc(clickstream):
-    sf = dd.from_pandas(clickstream, npartitions=10) \
-        .map_partitions(
-        sparse_one_hot,
-        categories={'page_id': list('ABCDE')},
-        meta=list
-    )
-
+    sf = one_hot_encode(dd.from_pandas(clickstream, npartitions=10),
+                        categories={'page_id': list('ABCDE'),
+                                    'other_categorical': list('FGHIJ')},
+                        index_col=['index', 'id'])
     res = sf.loc['2016-01-15':'2016-02-15']
-    res = sp.SparseFrame.concat(res.compute(get=get_sync).tolist())
-    assert res.index.date.max() == dt.date(2016, 2, 15)
-    assert res.index.date.min() == dt.date(2016, 1, 15)
+    res = res.compute(get=get_sync)
+    assert res.index.levels[0].max().date() == dt.date(2016, 2, 15)
+    assert res.index.levels[0].min().date() == dt.date(2016, 1, 15)
```

Comment:
No need to use datetime if you already need/have pandas, but that's so lightweight I don't think it justifies changing it.